### PR TITLE
ref #245: Cleanup

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#245
| License       | MIT

Just cleanup with close to no functional changes.

### CODE REVIEW

File: UPGRADE-6.3.md [hunk 1]
```diff
@@ -220,6 +220,103 @@ German date format. For backwards compatibility reasons they work with German da
 - Changed method `WriteLogEntry()`: parameter `$sLogFileName` is now ignored.
 - Changed method `WriteLogEntrySimple()`: parameter `$sLogFileName` is now ignored.
 
+## Backend Theme Library
+
+The Backend was upgraded to Bootstrap 4.1.3.
+
+See the [Bootstrap Migration Guide](https://getbootstrap.com/docs/4.1/migration/) for needed changes to your backend modules.
```
Comment (UlrichKu) [inline UPGRADE-6.3.md]:
```suggestion
See the [Bootstrap Migration Guide](https://getbootstrap.com/docs/4.1/migration/) for required changes to your backend modules.
```

File: UPGRADE-6.3.md [hunk 2]
```diff
@@ -220,6 +220,103 @@ German date format. For backwards compatibility reasons they work with German da
...
+
+During the upgrade to Bootstrap 4 the following styles where checked and these are our findings:
```
Comment (UlrichKu) [inline UPGRADE-6.3.md]:
```suggestion
During the upgrade to Bootstrap 4 the following styles were checked and these are our findings:
```

Comment (UlrichKu) [inline UPGRADE-6.3.md]: What kind of findings?

Something exists in these locations? Or was changed? To where?

findings -> changes?

File: UPGRADE-6.3.md [hunk 3]
```diff
@@ -220,6 +220,103 @@ German date format. For backwards compatibility reasons they work with German da
...
+- hidden/visible-xs, sm, md, lg
+- label
+- navbar-form
+- navbar-btn
+- progress-bar*
+
+## Backend JQuery
+
+JQuery that is used in the Chameleon backend was upgraded to version 3.3.1. For backwards compatibility
+jquery.migrate 1.4.1 was added, but will be removed in a future Chameleon version.
```
Comment (UlrichKu) [inline UPGRADE-6.3.md]: But all tasks printed there are done for the backend?

File: UPGRADE-6.3.md (hunk 1 above)
Comment (umulmrum) [reply UPGRADE-6.3.md]: Changed.

File: UPGRADE-6.3.md (hunk 2 above)
Comment (umulmrum) [reply UPGRADE-6.3.md]: Changed.

Comment (umulmrum) [reply UPGRADE-6.3.md]: Changed wording.

File: UPGRADE-6.3.md (hunk 3 above)
Comment (umulmrum) [reply UPGRADE-6.3.md]: This is to be read in the context of the whole document, so we need to make clear for every section that it only affects the backend.

File: UPGRADE-6.3.md (hunk 2 above)
Comment (UlrichKu) [reply UPGRADE-6.3.md]: There is currently only one commit.

**Comment by umulmrum** [comment]
Also reordered new sections of UPGRADE-6.3md that were added in the meantime.

**Pull request merged**